webapp/graphite/finders/standard.py

```diff
@@ -3,6 +3,14 @@
 from os.path import isdir, isfile, join, basename
 from django.conf import settings
 
+try:
+    from os import scandir
+    del scandir
+    del os
+    import os as scandir
```

Comment:
I'm not sure why we need 
```
+    del scandir
+    del os
+    import os as scandir
```

Maybe add a comment ?

The canonical way to import scandir seems to be:
```
# Use the built-in version of scandir/walk if possible, otherwise
# use the scandir module version
try:
    from os import scandir, walk
except ImportError:
    from scandir import scandir, walk
```